src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV1Spec.scala

```diff
@@ -772,7 +754,7 @@ class ResourceRoutesV1Spec extends FlatSpec with Matchers with ScalatestRouteTes
 
   it should "404 when deleting a resource that exists but can't be seen by the user" in {
     val resourceType = ResourceType(ResourceTypeName("rt"), Set(SamResourceActionPatterns.alterPolicies, SamResourceActionPatterns.readPolicies), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
-    val samRoutes = createSamRoutes(Map(resourceType.name -> resourceType))
```

Comment:
createSamRoutes is only used in this testcase (in this file) and it doesn't seem to be necessary for the test to pass.